write-fonts/src/tables/variations.rs

```diff
@@ -39,3 +270,77 @@ impl<'a> FromObjRef<Option<read_fonts::tables::variations::Tuple<'a>>> for Vec<F
             .unwrap_or_default()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn point_pack_words() {
+        let thing = PackedPointNumbers {
+            is_all: false,
+            numbers: vec![1002, 2002, 8408, 12228],
+        };
+
+        let runs = thing.iter_runs().collect::<Vec<_>>();
+        assert!(runs[0].are_words);
+        assert_eq!(runs[0].last_point, 0);
+        assert_eq!(runs[0].points, &[1002, 2002, 8408, 12228]);
```

Comment:
Suggest:

```rust
assert_eq!(PackedPointRun{ are_words: true, last_point: 0, etc }, runs);
```

Gives a nicer diff and has it built in to confirm you didn't get 2+ runs.

Reply:
I tried it like this but it ends up being more verbose and also harder to debug, since it's harder to track down the actual value contributing to a failure.